package unit12.exceptions.day46.exceptionexamples;

public class ExceptionExample {
    public static void main(String[] args) {
        System.out.println(10/0);
        System.out.println("---After the exception---");
    }
}
